dist/leaflet.distortableimage.js

```diff
@@ -1139,6 +1139,59 @@ L.LockHandle = L.EditHandle.extend({
 	_onHandleDrag: function() {
 	},
 
+  _bindListeners: function() {
+    this.on(
+      {
+        dragstart: this._onHandleDragStart,
+        drag: this._onHandleDrag,
+        dragend: this._onHandleDragEnd,
+        mouseover: this._setTooltip
+      },
+      this
+    );
+
+    this._handled._map.on("zoomend", this.updateHandle, this);
+
+    this._handled.on("update", this.updateHandle, this);
+  },
+
+  _setTooltip: function () {
+    
+	  var Icon = L.Icon.extend({
+      options: {
+        iconSize: [ 50, 50 ],
+        iconAnchor: [ 25, 50 ]
+      }
+    });
+   
+	  var question = new Icon({
+      iconUrl: '../assets/icons/markers/question-mark.png'
+    });
+	  
+	  this._questionMark = new L.marker(this._handled.getCorner(this._corner), { icon: question })
+		  .bindTooltip("Locked!")
+		  .addTo(this._handled._map)
+		  .openTooltip();
+	  
+	  setTimeout(function () {
+		  var el = [];
+		  var marks = Array.from(document.querySelectorAll(".leaflet-interactive"));
+		  for (var i = 0; i < marks.length; i++) { 
+			  if (/question/.test(marks[ i ].currentSrc)) { 
+				  el.push(marks[ i ]);
+			  }
+		  }
+		  var tt = document.querySelector(".leaflet-tooltip");
```

Comment:
Be careful with single and double letter variables ;)

Reply:
Sure! 😄 👍